src/global.d.ts

```diff
@@ -3,19 +3,19 @@
 declare namespace GlobalMixins
```

Comment:
hey @danielbarion I believe this should be `PixiMixins` for v8

Reply:
@Zyie like that?

![image](https://github.com/user-attachments/assets/5f33587b-55d3-47d6-9eb6-9c6ec6d74e5f)

Reply:
@Zyie looks like @davidfig didn't see this thread, but please let me know what the correct types is and I'll create a PR and merge for 6.0.1

Reply:
hey @danielbarion yeah this looks correct